Kickstarter-iOS/Views/Controllers/SettingsViewController.swift

```diff
@@ -110,7 +110,7 @@ final class SettingsViewController: UIViewController {
 
   private func goToAppStore(link: String) {
     guard let url = URL(string: link) else { return }
-    UIApplication.shared.openURL(url)
+    UIApplication.shared.open(url)
```

Comment:
I've tried searching the code base for `openURL(` and found `UIApplication.shared.openURL(url)` in `Library > SafariActivity.swift:perform()` which now causes compiler warning. Wanna take a look at it?